### PR TITLE
Fix invalid prop types and invalid keys.

### PR DESCRIPTION
**Pull request by Hyperkid123**
Fixes multiple react warnings at runtime

A possible breaking change is replacing Fragment with span to remove the invalid prop warning. I have tried clicking in the chrome global filter and it was working. But if someone could double-check it would be grand.

### CODE REVIEW

File: packages/charts/src/SeverityLine/SeverityLine.js [hunk 1]
```diff
@@ -45,9 +45,9 @@ const SeverityLine = ({ title, value, className, tooltipMessage, config, chartPr
 
 SeverityLine.propTypes = {
     value: PropTypes.number.isRequired,
-    title: PropTypes.string.isRequired,
+    title: PropTypes.oneOfType([ PropTypes.node, PropTypes.arrayOf(PropTypes.node) ]).isRequired,
```
Comment (rvsia) [inline packages/charts/src/SeverityLine/SeverityLine.js:48]: Isn't PropTypes.node the same as an array of nodes?

Comment (Hyperkid123) [reply packages/charts/src/SeverityLine/SeverityLine.js:48]: Yes, it is. Nasty habit of mine :+1:

**Review by rvsia — APPROVED** [review]

**Review by karelhala — APPROVED** [review]

**Comment by rvsia** [comment]
Oh, you have to update PF version to satisfy DDF needs! 😅

**Comment by Hyperkid123** [comment]
@rvsia the slider component is not released in LTS of PF. I've locked ddf to prev version. You can fix it if you want but if you do scope the PF version only to sources package pls.

**Pull request merged**